sys/include/shell.h

```diff
@@ -43,6 +50,40 @@ extern "C" {
 #  endif
 #endif
 
+/**
+ * @brief Set to 1 to disable shell's echo
+ */
+#ifndef CONFIG_SHELL_NO_ECHO
+#define CONFIG_SHELL_NO_ECHO 0
+#endif
+
+/**
+ * @brief Set to 1 to disable shell's prompt
+ */
+#ifndef CONFIG_SHELL_NO_PROMPT
+#define CONFIG_SHELL_NO_PROMPT 0
+#endif
+
+/**
+ * @brief Set to 1 to disable shell's echo
+ * @deprecated This has been replaced by @ref CONFIG_SHELL_NO_ECHO and will be
+ *             removed after release 2021.07.
+ */
+#ifndef SHELL_NO_ECHO
+#define SHELL_NO_ECHO 0
```

Comment:
Would it make sense to set this to
```suggestion
#define SHELL_NO_ECHO CONFIG_SHELL_NO_ECHO
```
?